signalfx-java/src/main/java/com/signalfx/metrics/connection/AbstractHttpDataPointProtobufReceiverConnection.java

```diff
@@ -63,20 +69,31 @@ protected abstract HttpEntity getEntityForVersion(
             List<SignalFxProtocolBuffers.DataPoint> dataPoints);
 
     @Override
-    public void backfillDataPoints(String auth, String source, String metric,
-                                   List<SignalFxProtocolBuffers.Datum> datumPoints)
+    public void backfillDataPoints(String auth, String metric, String metricType, String orgId, Map<String,String> dimensions,
+                                   List<SignalFxProtocolBuffers.PointValue> datumPoints)
             throws SignalFxMetricsException {
         if (datumPoints.isEmpty()) {
             return;
         }
+
+        List<NameValuePair> params = Lists.newArrayList();
+        params.add(new BasicNameValuePair("orgid", orgId));
+        params.add(new BasicNameValuePair("metric_type", metricType));
+        params.add(new BasicNameValuePair("metric", metric));
+
+        // Each dimension is added as a param in the form of "sfxdim_DIMNAME"
+        for(Map.Entry<String,String> entry : dimensions.entrySet()) {
```

Comment:
Space before open paren. Make sure you setup our IntelliJ Java style (from `editor/` in our `dtools` repo).

Reply:
Done and thanks for the tip. That should prevent future problems. :)